src/components/ImageCarousel.tsx

```diff
@@ -40,7 +54,7 @@ const ImageCarousel = ({ images }: ImageCarouselPropsType): ReactElement => {
     return !currentIndex || !allSize ? (
       <></>
     ) : (
-      <PaginationView>
+      <PaginationView topPosition={minimized ? -10 : 10}>
```

Comment:
Better to just pass `minimized` here imo and do the styling in the styled component